tests/integration/modules/beacons.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+
+# Python Libs
+from __future__ import absolute_import
+
+# Salt Libs
+from salt.modules import beacons
+import integration
+from salttesting import skipIf
+from salttesting.helpers import destructiveTest, ensure_in_syspath
+
+ensure_in_syspath('../../')
+
+beacons.__opts__ = {}
+
+
+class BeaconsTest(integration.ModuleCase):
+    '''
+    Tests the beacons execution module
+    '''
+    @destructiveTest
```

Comment:
If all of the tests in this class are destructive, then why not put that decorator as the top of the test class?